app/controllers/v2/link_sets_controller.rb

```diff
@@ -23,6 +27,12 @@ def get_linked
 
   private
 
+    def with_drafts?
+      # Cast the `with_drafts` query param to a real boolean, and default to
+      # `true` to preserve existing behaviour
+      ActiveModel::Type::Boolean.new.cast(params.fetch(:with_drafts, true))
```

Comment:
Is the type case done as ?with_drafts=false is interpreted as true (being a string)?